### PR TITLE
Alex/6 create a docker compose development environment

### LINKED ISSUE #6: create a docker-compose development environment
We need to use docker-compose for development and testing so that other developers can contribute without extensive configurations of their systems.

### PR DESCRIPTION
**Pull request by aleksandr-efimenko**
- Update the docker-compose file
- Fix issues with running Prisma
- Modify reses_db.sh file to reset and create db from the Prisma models

Closes #6

### CODE REVIEW

File: compose/reset_db.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+docker-compose stop db
+docker-compose rm -f db
+docker-compose up -d db
+sleep 15
+docker-compose exec -t web npx prisma db push #command to migrate using prisma
```
Comment (cohenoshriconsulting) [inline compose/reset_db.sh:5]: You will need to rebuild your web docker image before it will be able to use the Prisma schema

**Review by cohenoshriconsulting — APPROVED** [review]

File: compose/reset_db.sh (hunk 1 above)
Comment (cohenoshriconsulting) [reply compose/reset_db.sh:5]: docker-compose rm -f web

Comment (cohenoshriconsulting) [reply compose/reset_db.sh:5]: docker-compose up -d --parallel

Comment (aleksandr-efimenko) [reply compose/reset_db.sh:8]: Added rebuild of the web container

**Pull request merged**